code/modules/mob/living/silicon/silicon.dm

```diff
@@ -66,14 +66,16 @@
 /mob/living/silicon/drop_item()
 	return
 
-/mob/living/silicon/emp_act(severity)
+/mob/living/silicon/robot/emp_act(severity)

```

Comment:
this should be untouched; you should make a `/mob/living/silicon/robot/emp_act(severity)` and have it call to parent, then do different things based on severity.